src/abs.ott

```diff
@@ -255,6 +321,70 @@ defn
   ---------------------------- :: var
   F1 ... Fn , s |- x ~> s |- t
 
+  (-) z = z'
+  ---------------------------------------------------- :: neg
+  F1 ... Fn , s |- - z ~> s |- z'
+
+  (!) b = b'
+  ---------------------------------------------------- :: not
+  F1 ... Fn , s |- ! b ~> s |- b'
+
+  z1 (+) z2 = z
+  ---------------------------------------------------- :: add
+  F1 ... Fn , s |- z1 + z2 ~> s |- z
+
+  z1 (*) z2 = z
+  ---------------------------------------------------- :: mul
+  F1 ... Fn , s |- z1 * z2 ~> s |- z
+
+  z1 (==) z2 = b
+  ---------------------------------------------------- :: eq
+  F1 ... Fn , s |- z1 == z2 ~> s |- b
+
+  z1 (<) z2 = b
+  ---------------------------------------------------- :: lt
+  F1 ... Fn , s |- z1 < z2 ~> s |- b
```

Comment:
Why not skip the `b` and put the boolean operation directly as the result?

Reply:
That is what I wanted, but I could not get it to work with Ott because I was (mistakenly) trying to define the operations under formulas instead of under z_ and b_.

Thanks!